utilities/mobile_server_pool.py

```diff
@@ -63,9 +69,14 @@ def get_nodes_from_pool_server(num_of_nodes, nodes_os_type, node_os_version, job
     :return: list of nodes reserved
     """
     # Getting list of available nodes through n1ql query
+    query_phone_in_slave = ""
+    if nodes_os_type in MOBILE_OS:
+        if slave_ip is None:
+            raise Exception("\n**** we need slave ip to get phone info on that slave ***")
+        query_phone_in_slave = "AND slave_ip='{}'".format(slave_ip)
     query_str = "select meta().id from `{}` where os='{}' " \
```

Comment:
There are 5 '{}' but 4 variables (BUCKET_NAME, nodes_os_type, node_os_version, query_phone_in_slave), is that OK?